src/odemis/util/angleres.py

```diff
@@ -427,3 +435,166 @@ def _CreateMirrorMask(data, pixel_size, pole_pos, offset_radius=0, hole=True):
 
     return circle_mask
 
+
+def _CreateMirrorMask_Rectangular(data, theta_indices, phi_indices, hole=True):
+    """
+    Creates the mask (i.e. True inside half circle, False outside) based on
+    parabola_f and focus_distance values in theta/phi coordinates.
+    :param data: (DataArray) The data array containing the image. Shape is (x, y)
+    :param theta_indices : (ndarray) 1D array of the theta values,
+                           containing the corresponding theta value for each px of the input data in y.
+    :param phi_indices: (ndarray) 1D array of the phi values,
+                            containing the corresponding phi value for each px of the input data in x.
+    :param hole: (bool) Crop the area around the pole (hole in mirror for the
+                        ebeam to pass through) if True.
+    :return: (boolean ndarray) Mask for rectangular representation. Shape is (theta, phi).
+    """
+
+    # solve equality ar^2-1/(4a)=x.
+    # c=1/(2*(a*cos(phi)*sin(theta)+sqrt(a^2*(cos(theta)^2+(cos(phi)^2+sin(phi)^
+    # 2)*sin(theta)^2)))); The cos(phi)^2+sin(phi)^2=1 so we can omit that. Than
+    # we have cos(theta)^2+sin(theta)^2 which also drops out. That leaves the
+    # square root of a^2
+
+    # TODO @Toon how can we clean up this to match the odemis vars
+    xmax = data.metadata.get(model.MD_AR_XMAX, AR_XMAX)
+    hole_diameter = data.metadata.get(model.MD_AR_HOLE_DIAMETER, AR_HOLE_DIAMETER)
+    focus_distance = data.metadata.get(model.MD_AR_FOCUS_DISTANCE, AR_FOCUS_DISTANCE)
+    parabola_f = data.metadata.get(model.MD_AR_PARABOLA_F, AR_PARABOLA_F)
+
+    a = 0.1
```

Comment:
Hopefully, comparing these values to the default values + the code in `_CreateMirrorMask()` should hint enough Toon...